hack/build/build-via-appstudio.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+SCRIPTDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )" 
+
+
+if [ -z "$MY_QUAY_USER" ]; then
+  echo "Missing MY_QUAY_USER variable"
+  exit 1
+fi
+
+if ! grep -q quay.io ~/.docker/config.json; then
```

Comment:
This did not work for me on Fedora 35.

I needed to use `${XDG_RUNTIME_DIR}/containers/auth.json`

Reply:
Also having Fedora 35 and it's under `~/.docker/config.json` which is fallback, primary is that `${XDG_RUNTIME_DIR}/containers/auth.json`, updated the script with the file detection